src/annotator/guest.js

```diff
@@ -499,21 +503,21 @@ export default class Guest {
     this._updateAnchors(this.anchors.concat(anchors), true /* notify */);
 
     // Let other frames (eg. the sidebar) know about the new annotation.
-    this._annotationSync.sync([annotation]);
+    this._sidebarRPC.call('syncAnchoringStatus', annotation);
```

Comment:
Changed this RPC method to send a single (safe) annotation, instead of an array.